RoddyCore/src/de/dkfz/roddy/execution/jobs/JobManager.groovy

```diff
@@ -33,23 +33,15 @@ class JobManager {
         return RuntimeService._createJobName(baseFile.executionContext, baseFile, toolID, reduceLevel)
     }
 
-    // Backward compatibility issue. Should be static
-//    String createJobName(BaseFile baseFile, String toolID, boolean reduceLevel) {
-//        return RuntimeService._createJobName(baseFile.executionContext, baseFile, toolID, reduceLevel)
-//    }
-
     Command createCommand(GenericJobInfo jobInfo) {
-        jobManager.createCommand(jobInfo)
+        BEJob job = new Job(jobInfo)
+        jobManager.createCommand(job, jobInfo.tool, jobInfo.parentJobIDs)
```

Comment:
Could be shortened to either
createCommand(jobInfo) or
createCommand(job)

Reply:
Well, the whole class was not set `CompileStatic` and most of its code -- including this method -- is not used anywhere in the AQCWF, SNV, ACEseq, Indel plugins. I decided to prune this class.

Now only two methods are left:
```groovy
    static String createJobName(BaseFile baseFile, String toolID, boolean reduceLevel) {
        return RuntimeService._createJobName(baseFile.executionContext, baseFile, toolID, reduceLevel)
    }
    
    boolean executesWithoutJobSystem() {
        return jobManager.executesWithoutJobSystem()
    }
```

I suggest we remove the JobManager class completely.

What do you think?